### ISSUE TITLE
LabelPlus support

### ISSUE THREAD

**Issue by robsonsobral** [omaralvarez/deluge-autoremoveplus#20]
Hi! Thank you for the plugin.

Is it possible to support the labels created using [LabelPlus](https://github.com/ratanakvlun/deluge-labelplus)?

Thank you!

**Comment by omaralvarez** [comment]
Yes, it should be possible. Although this feature will have to wait for a while, because I am working on a redesign of the plugin with a lot of new features. I will certainly consider this for future releases.

**Comment by robsonsobral** [comment]
Thank you!

Both plugins will make a perfect automation setup.

Thank you for your attention.

**Comment by KyleSanderson** [comment]
Any update on this? I'm trying to migrate from my now decade old rtorrent setup.

**Comment by omaralvarez** [comment]
Not yet ready. I have been up to my neck in work and several other projects. I will give this some love in the coming months as soon as I have time. Sorry for the delay guys!

**Comment by Neaox** [comment]
Any updates?
